adders/prettier/config/adder.ts

```diff
@@ -0,0 +1,128 @@
+import { dedent, defineAdderConfig } from "@svelte-add/core";
+import { options } from "./options.js";
+
+export const adder = defineAdderConfig({
+    metadata: {
+        id: "prettier",
+        name: "Prettier",
+        description: "An opinionated code formatter",
+        environments: { svelte: true, kit: true },
+        website: {
+            logo: "./prettier.svg",
+            keywords: ["prettier", "code", "formatter", "formatting"],
+            documentation: "https://prettier.io",
+        },
+    },
+    options,
+    integrationType: "inline",
+    packages: [
+        { name: "prettier", version: "^3.3.2", dev: true },
+        { name: "prettier-plugin-svelte", version: "^3.2.5", dev: true },
+        {
+            name: "eslint-config-prettier",
+            version: "^9.1.0",
+            dev: true,
+            condition: ({ dependencies }) => hasEslint(dependencies),
+        },
+    ],
+    files: [
+        {
+            name: () => `.prettierignore`,
+            contentType: "text",
+            content: ({ content }) => {
+                if (content) return content;
+                return dedent`
+                    # Package Managers
+                    package-lock.json
+                    pnpm-lock.yaml
+                    yarn.lock
+                `;
+            },
+        },
+        {
+            name: () => ".prettierrc",
+            contentType: "json",
+            content: ({ data }) => {
+                data.useTabs ??= true;
+                data.singleQuote ??= true;
+                data.trailingComma ??= "none";
+                data.printWidth ??= 100;
+                data.plugins ??= [];
+                data.overrides ??= [];
+
+                const plugins: string[] = data.plugins;
+                if (!plugins.includes("prettier-plugin-svelte")) {
+                    data.plugins.unshift("prettier-plugin-svelte");
+                }
+
+                const overrides: { files: string | string[]; options?: { parser?: string } }[] = data.overrides;
+                const override = overrides.find((o) => o?.options?.parser === "svelte");
+                if (!override) {
+                    overrides.push({ files: "*.svelte", options: { parser: "svelte" } });
+                }
+            },
+        },
+        {
+            name: () => "package.json",
+            contentType: "json",
+            content: ({ data, dependencies }) => {
+                data.scripts ??= {};
+                const scripts: Record<string, string> = data.scripts;
+                const CHECK_CMD = "prettier --check .";
+                scripts["format"] ??= "prettier --write .";
+                scripts["format:check"] ??= CHECK_CMD;
```

Comment:
Do we need that? The current `create-svelte` is not adding that command, and you are able to run that by running the `lint` script
[-eslint+prettier](https://github.com/sveltejs/kit/blob/main/packages/create-svelte/shared/-eslint%2Bprettier/package.json)
[+eslint-prettier](https://github.com/sveltejs/kit/blob/main/packages/create-svelte/shared/%2Beslint-prettier/package.json)
[+eslint+prettier](https://github.com/sveltejs/kit/blob/main/packages/create-svelte/shared/%2Beslint%2Bprettier/package.json)

Reply:
fair point! will address it